gooddata-pandas/gooddata_pandas/dataframe.py

```diff
@@ -17,7 +17,7 @@
     make_pandas_index,
 )
 from gooddata_sdk import Attribute, BareExecutionResponse, ExecutionDefinition, Filter, GoodDataSdk
-from gooddata_sdk.compute.model.execution import ResultSize
+from gooddata_sdk.compute.model.execution import ResultCacheMetadataWrapper, ResultSizeBytes, ResultSizeDimensions
```

Comment:
If I am not mistaken you added these classes in gooddata-sdk/gooddata_sdk/__init__.py meaning you can use `from gooddata_sdk` to import them.

Reply:
fixed